core/formats/columnstore2.hpp

```diff
@@ -82,45 +82,46 @@ class column final : public irs::column_output {
 
   class address_table {
    public:
+    address_table(ManagedTypedAllocator<uint64_t> alloc) : alloc_{alloc} {
+      offsets_ = alloc_.allocate(kBlockSize);
+      offset_ = offsets_;
+    }
+
+    ~address_table() { alloc_.deallocate(offsets_, kBlockSize); }
```

Comment:
Also deque with columns works better now:
blocks in deque could handle few columns and we don't store this memory forever.
only while write segment